esp/xslt/wsecl3_form.xsl

```diff
@@ -311,7 +311,7 @@ function switchInputForm()
                      </span>
               </td>
             </tr>
-            <xsl:if test="$includeRoxieOptions=1">
+            <xsl:if test="$includeRoxieOptions=1 or $includeRoxieOptions=2">
```

Comment:
Should this be != 0?  Currently 3 is not included.

Reply:
The 3 is not included here because it is for a roxie which always runs a published query as a workunit. The 'Save stats to workunit.' option and 'Get summary stats.' option should not give to that roxie. @afishbeck please comment.

Reply:
I asked because the code on line 348 will never be processed as it stands.

Also, is the traceLevel check-box still relevant?

Reply:
The includeRoxieOptions will be set to 3 in the code of esp/services/ws_ecl/ws_ecl_service.cpp for a workunit roxie. In that case, the line 348 will be processed. 

> Also, is the traceLevel check-box still relevant?
I am not sure. @afishbeck please advise.

Reply:
Yes, when calling a published query you can tell Roxie the traceLevel to use.  I’m not sure if there is an equivalent work unit parameter.  As it is now that shouldn’t be exposed when submitting a work unit.